trio/_core/_mock_clock.py

```diff
@@ -124,56 +106,39 @@ def autojump_threshold(self):
     @autojump_threshold.setter
     def autojump_threshold(self, new_autojump_threshold):
         self._autojump_threshold = float(new_autojump_threshold)
-        self._maybe_spawn_autojump_task()
-        if self._autojump_cancel_scope is not None:
```

Comment:
Suggest you remove from `__init__` the initializations of `_autojump_task` and `_autojump_cancel_scope`, since those aren't used anymore.